parametric/utils/headers.py

```diff
@@ -0,0 +1,13 @@
+from parametric.protos.apm_test_client_pb2 import DistributedHTTPHeaders
+
+
+def make_single_request_and_get_headers(test_library, headers_list):
```

Comment:
```suggestion
def make_single_request_and_get_inject_headers(test_library, headers_list):
```

maybe a tad more clear?

Reply:
Good suggestion, implemented in d45c213f8ba18734c1e28faa30ff220956726e7e